src/components/MultiSelect/index.js

```diff
@@ -147,20 +141,19 @@ const MultiSelect = React.forwardRef((props, ref) => {
         }
     };
 
-    const handleOutsideClick = event => {
-        if (
-            event.target !== triggerRef.current.buttonRef.current &&
-            !triggerRef.current.buttonRef.current.contains(event.target)
-        ) {
-            // eslint-disable-next-line no-use-before-define
-            stopListeningOutsideClick();
-            setIsOpen(false);
-        }
-    };
-    const [startListeningOutsideClick, stopListeningOutsideClick] = useOutsideClick(
+    useOutsideClick(
         dropdownRef,
-        handleOutsideClick,
+        event => {
+            if (
+                event.target !== triggerRef.current.buttonRef.current &&
+                !triggerRef.current.buttonRef.current.contains(event.target)
+            ) {
+                setIsOpen(false);
+            }
+        },
+        isOpen,
```

Comment:
Multiselect outside click is not working, we should add integration test in order to avoid this happening